src/ucc/api/ucc.h

```diff
@@ -160,7 +160,8 @@ typedef enum {
     UCC_COLL_TYPE_GATHER             = UCC_BIT(6),
     UCC_COLL_TYPE_SCATTER            = UCC_BIT(7),
     UCC_COLL_TYPE_FANIN              = UCC_BIT(8),
-    UCC_COLL_TYPE_FANOUT             = UCC_BIT(9)
+    UCC_COLL_TYPE_FANOUT             = UCC_BIT(9),
+    UCC_COLL_TYPE_ALLTOALLV          = UCC_BIT(10)
```

Comment:
generic question: do we want any particular "fancier" ordering of this enum ? like alphabetical .. just a thought (looking for smth nice to read ))